src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/tests/testdata/adu-mqtt-client-module-test-data/du-config.json

```diff
@@ -0,0 +1,71 @@
+{
+    "schemaVersion": "2.0",
+    "aduShellTrustedUsers": [
+      "adu"
+    ],
+    "manufacturer": "contoso",
+    "model": "espresso-v1",
+    "agents": [
+      {
+        "name": "main",
+        "runas": "adu",
+        "connectionSource": {
+          "connectionType": "MQTTBroker",
+          "connectionData": {
+              "dps": {
+                "globalDeviceEndpoint" : "global.azure-devices-provisioning.net",
+                "idScope" : "0ne00ABF077",
+                "registrationId" : "contoso-blue-device03-devbox-wus3",
+                "apiVersion" : "2021-06-01",
+                "tcpPort" : 8883,
+                "useTLS": true,
+                "mqttVersion" : 4,
+                "cleanSession" : true,
+                "keepAliveInSeconds" : 3600,
+                "clientId" : "contoso-blue-device03-devbox-wus3",
+                "certFilex" : "/home/contoso/aducerts/contoso-blue-device03-devbox-wus3.cer",
```

Comment:
will test pass if these certs aren't at this location?  same for lines 38, 39

Reply:
No, as these tests are not intended for automation at the moment.